platformcomponents/desktop/button.json

```diff
@@ -18,7 +18,8 @@
       },
       "#disabled": {
         "background": "@theme-button-secondary-normal",
-        "text": "@theme-text-primary-disabled"
+        "text": "@theme-text-primary-disabled",
+        "border": "@theme-button-border-disabled"
```

Comment:
as well as here; we do not have these border in other non-hc themes' ghost-disabled button

Reply:
Same issue as `button-primary`. We need to define it because we use it now, just `transparent` for non-HC themes.

Reply:
If you need a different border for HC theme, then why not overwriting it in under platformcomponent/HC ?